docs/_static/auth/iam.yaml

```diff
@@ -456,11 +456,24 @@ components:
         scope:
           type: string
           description: granted scopes
-        vcs:
+        pdp_map:
```

Comment:
mixing short form and long form now, gotta choose one?

Reply:
you could take a look at https://www.iana.org/assignments/jwt/jwt.xhtml for ideas. `verified_claims` might be a good fit?

Reply:
I don't think `verified_claims` is a good fit (or any other registered claim name). It comes from the OpenID Connect realm and is focussed on user identity. The required `trust_framework` field per claims object also adds unnecessary complexity IMO. I prefer to reserve this for user identity as it also provides a standardized output on the assurance level of the claims about the user.

We'd probably have to customize the claim for our purpose so I prefer to just use a custom claim. This is an internal endpoint so a custom claim does not impact interoperability. I've changed the name to `input_descriptor_constraint_id_map`. Very verbose, but I am open to changes